philip/src/App.svelte

```diff
@@ -1,31 +1,27 @@
 <script>
-  import Tab from "./Tab.svelte";
+  import { routes, curRoute, active_tab } from "./store";
+  import { Navigation, Footer } from "./components";
+  import Router from "./Router.svelte";
+
+  export let name;
 
   let session = {};
-  let active_tab;
   let pgi = pubgate_instance;
 
-  if(session.user) {
-      active_tab = 'inbox'
-  } else {
-    active_tab = pgi ? "local" : "about";
-  }
-
-  function selectTab(event) {
-    event.preventDefault();
-
-    active_tab = this.href.split("#")[1];
+  if (session.user) active_tab.set("inbox");
+  else active_tab.set(pgi ? "local" : "about");
 
+  function selectTab(target) {
+    const active_tab = target.pathname;
```

Comment:
Same timeline loads issue
active_tab from outside of the selectTab function should be updated here too